packages/playground-server/src/types.ts

```diff
@@ -48,7 +61,7 @@ export type GetAppsResponseData = {
 };
 
 export type PlaygroundUserData = {
-  email: string;
+  email?: string;
```

Comment:
shouldn't be optional

Reply:
Fixed via 3bdd92d. Matchmaking now has its own `MatchmakeUserData` type.